routes/index.js

```diff
@@ -191,11 +191,30 @@ module.exports = function (passport) {
 
 
   router.get('/userManagement', checkAuth, function (req, res) {
-    dbAPI.getUsers().then(function (ret) {
-      res.render('userManagement.njk', { obj: ret, authenticated: true });
-    });
-  });
 
+    Promise.all([
+      dbAPI.getUsers(undefined),
+      dbAPI.getStations(true)
```

Comment:
Since we are calling stations "Online" and "Offline", I feel like the dropdown should display both and not just "Online" stations. The same goes for the other route handler. Just don't pass anything into `getStations()`

Reply:
I guess I can pass undefined instead of leaving it empty.

Reply:
Passing nothing is the same as passing undefined I believe.